types/rosbag/index.d.ts

```diff
@@ -0,0 +1,508 @@
+// Type definitions for rosbag 2.6
+// Project: https://github.com/cruise-automation/rosbag.js#readme
+// Definitions by: Itthipol Lateh <https://github.com/Rovula>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.7
```

Comment:
I'd say you no longer need this on DT (this is current version enforced/tested on DT, after TS 4.4 release), unless there was a specific reason of adding this one:
```suggestion
```